### DISCUSSION TITLE
FormattedPlural

### DISCUSSION THREAD

**Discussion by dejurin** [amannn/next-intl#177] [Q&A] [answered]
How about support this 

`<FormattedPlural value={10} one="message" other="messages" />`

or something like?

I have a problem where I take the values of the plural options from the database.

**Comment by dejurin** [comment, marked as answer]
I get it, I apologize. For the new topic.
I didn't think it would work.
Maybe I should add this as an example.

```JSON
{"msg": "Ви маєте {count, plural, =0 {{zero}} =1 {{one}} other {{other}}} повідомлень."}
```

```JS
t('msg', { num: 2, zero: 'нуль', one: 'одне', other: 'багато' })
```

```JS
// Ви маєте багато повідомлень.
```

**Reply by amannn** [reply]
You should be able to embed the placeholders directly btw.:

```js
{"msg": "Ви маєте {count, plural, =0 {нуль} =1 {одне} other {багато}} повідомлень."}
```

```js
t('msg', { num: 2})
```
